src/Monitor/Monitor/OutputClasses/PSMetricTabularResult.cs

```diff
@@ -13,6 +13,7 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Management.Monitor.Models;
+using Microsoft.Azure.Management.Monitor.Version2018_09_01.Models;
```

Comment:
This seems to be unnecessary.

Reply:
It's necessary since the Unit of the metric is referenced from Microsoft.Azure.Management.Monitor.Version2018_09_01.Models